tests/framework/PostProcessors/BasicStatistics/test_median_and_percentile.xml

```diff
@@ -0,0 +1,120 @@
+<?xml version="1.0" ?>
+<Simulation verbosity="debug">
+  <RunInfo>
+    <WorkingDir>median</WorkingDir>
+    <Sequence>sample,PP</Sequence>
+    <batchSize>1</batchSize>
+  </RunInfo>
+
+
+  <TestInfo>
+    <name>framework/PostProcessors/BasicStatistics/medianAndPercentile</name>
+    <author>ZHOUJ2</author>
+    <created>2020-10-08</created>
+    <classesTested>PostProcessors.BasicStatistics</classesTested>
+    <description>
+      This requirements test checks median calcuation and the percentile calcuation.
```

Comment:
Jia. The description is too minimal. Please, update it with something more detailed